design-tokens/$metadata.json

```diff
@@ -1,10 +1,13 @@
 {
   "tokenSetOrder": [
```

Comment:
Why are we exposing primitive tokens here?

Reply:
Because I've split the primitive and semantic tokens into their own folders, each theme needs to know what file to turn off and on.